example/config/README.md

```diff
@@ -0,0 +1,249 @@
+
+Available Imputation Rules: 
+- `mean`: The average value of the feature (for SpacetimeAggregation the mean is taken within-date).
+- `constant`: Fill with a constant value from a required `value` parameter.
+- `zero`: Fill with zero.
+- `null_category`: Only available for categorical features. Just flag null values with the null category column.
+- `binary_mode`: Only available for aggregate column types. Takes the modal value for a binary feature.
+- `error`: Raise an exception if any null values are encountered for this feature.
+
+    
+- `feature_aggregations`: 
+    - `prefix`: prefix given to the resultant tables
+    - `from_obj`: from_obj is usually a source table but can be an expression, such as a join (ie ```"cool_stuff join other_stuff using (stuff_id)"```)
+    - `knowledge_date_column`: The date column to use for specifying which records to include in temporal features. It is important that the column used specifies the date at which the event is known about, which may be different from the date the event happened.
+    - `aggregates_imputation`: top-level imputation rules that will apply to all aggregates functions can also specify `categoricals_imputation` or `array_categoricals_imputation`. You must specified at least one of the top-level or feature-level imputation to cover ever feature being defined.
+        - `all`: The `all` rule will apply to all aggregation functions, unless overridden by more specific one
+            - `type`: every imputation rule must have a `type` parameter, while some (like 'constant') have other required parameters (`value` here)
+            - `value`
+        - `max`: specifying `max` here will take precedence over the `all` rule for aggregations using a MAX() function
+    - `aggregates`: aggregates and categoricals define the actual features created. So at least one is required. Aggregates of numerical columns.
+        - (First quantity)
+            - `quantity`: Each quantity is a number of some and the list of metrics are applied to each quantity.
+            - `imputation`:  Imputation rules specified at the level of specific features will take precedence over the higer-level rules specified above. Note that the 'count' and 'sum' metrics will be imputed differently here.
+                - `count`:
+                    - `type`: `mean`
+                - `sum`:
+                    - `type`: `constant`
+                    - `value`: `137`
+            - `metrics`:
```

Comment:
perhaps a comment here that metrics/aggregation functions listed here will be used for the current aggregation only and must be defined separately for all aggregations